### ISSUE TITLE
Tables used for presentation in theme options aren't marked as such

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#1641]
### Description

On the theme options menus and on our cloning routine page, we present several sets of options within tables. These tables are used for presentation, but this information is not clearly indicated as such for users of assistive technology. The simplest fix would be to add `role="none"` to the various `<table>` elements to allow AT to disregard the tabular structure. In addition, we might also wish to replace tables altogether, dropping row header `<th>` elements with `<label>` elements.

**Comment by SteelWagstaff** [comment]
Step 1: Open issue with WP -- let them know that these tables are lacking the ARIA attribute they need.
Step 2: Test our proposed javascript "fix" in a screenreader.

**Comment by greatislander** [comment]
@SteelWagstaff There is an existing WordPress issue—can you ping me about this? I can get it on the accessibility meeting agenda for tomorrow.

**Comment by connerbw** [comment]
> Step 3: Test our proposed javascript "fix" in a screenreader.

Where we can't directly modify PHP/HTML (WordPress core), I Tested the following in NVDA 2019.1

```
// Accessibility fix on form tables
$( 'table.form-table' ).attr( 'role', 'none' );
```

Before: Table X, Row Y, blah blah blah
After: blah blah blah

Works for me.

**Comment by greatislander** [comment]
@connerbw @SteelWagstaff I'm going to be opening a fix for this in core, more to come.

**Comment by greatislander** [comment]
Ticket, with patch: https://core.trac.wordpress.org/ticket/46899

Should be fairly uncontroversial, as a similar patch was just approved and merged (https://core.trac.wordpress.org/ticket/46725).

**Comment by connerbw** [comment]
@greatislander we had a follow up meeting with Lisa and she said:

> The new role="presentation" in aria 1.1 is role="none" which is exactly the same but the newer version of the same thing

Your patch is using `role="presentation"` but we are switching to `role="none"` based on our follow up meeting.

**Comment by SteelWagstaff** [comment]
I think they're now considered synonyms, with some semantic web types having a preference for `none`: https://www.w3.org/TR/using-aria/#presentation

**Comment by greatislander** [comment]
I went with `rel="presentation"` as the other recent patch addressing the same issue (removing table semantics) did as well. But thanks for the additional info, @connerbw & @SteelWagstaff!

**Comment by mcgratay** [comment]
Table used for presentation on following pages still need role="none": 

- Publish
- Cover Generator
- Add New User 
- User Profile Page
- Sharing and Privacy Settings (from the Settings menu) 
- Export Settings (from the Settings menu) 
- Hypothesis Settings (from the Settings menu) 
- H5P Settings (from the Settings menu) 
![Screen Shot 2019-04-17 at 3 01 50 PM](https://user-images.githubusercontent.com/39064028/56314574-6fdcc600-6123-11e9-96e7-a29935cca484.png)

**Comment by dannylonglegs** [comment]
Dac and I figured out a way to make sure this doesnt happen again. thanks,
will fix!

**Comment by mcgratay** [comment]
Resolved 👍
